angular/Jenkinsfile

```diff
@@ -0,0 +1,100 @@
+#!groovy
+
+def tryStep(String message, Closure block, Closure tearDown = null) {
+    try {
+        block();
+    }
+    catch (Throwable t) {
+        slackSend message: "${env.JOB_NAME}: ${message} failure ${env.BUILD_URL}", channel: '#ci-channel', color: 'danger'
+
+        throw t;
+    }
+    finally {
+        if (tearDown) {
+            tearDown();
+        }
+    }
+}
+
+
+node {
+    stage("Checkout") {
+        checkout scm
+    }
+
+    stage('Test') {
+        tryStep "test", {
+            withCredentials([[$class: 'StringBinding', credentialsId: 'PASSWORD_EMPLOYEE', variable: 'PASSWORD_EMPLOYEE'],
+                             [$class: 'StringBinding', credentialsId: 'PASSWORD_EMPLOYEE_PLUS', variable: 'PASSWORD_EMPLOYEE_PLUS']]) {
+                sh "docker-compose -p atlas -f .jenkins/docker-compose.yml build && " +
```

Comment:
atlas -> predictive-parking?

P.S.: Volgensmij refereert dat naar de Docker compose naam. Dus die file hoort er dan ook bij. Misschien voor nu het hele Jenkins nog even niet in de eerste PR meenemen?